components/reddit/reddit.app.js

```diff
@@ -252,3 +252,17 @@ module.exports = {
     },
   },
 };
+  	  async getNewSubredditLinks(after_link, subreddit){
```

Comment:
Did you mean to push these changes to the Reddit branch?

Reply:
No, sorry.  But when I rebased that file had conflict so I had to resolved, and I committed. Was the best to do here? Stash them and not commit/push them?